src/lib/retrieveCollectionData.ts

```diff
@@ -18,6 +25,8 @@ export type CollectionDataTypeMap = {
   communities: ConvertToAttributeKeyedObjectWithNoBody<Community>
   'community-labels': ConvertToAttributeKeyedObjectWithNoBody<CommunityLabel>
   posts: ConvertToAttributeKeyedObject<Post>
+  events: ConvertToAttributeKeyedObject<CommunityEventCollectionData>
+  'event-hosts': ConvertToAttributeKeyedObject<CommunityEventHostCollectionData>
```

Comment:
```suggestion
  'community-event-hosts': ConvertToAttributeKeyedObject<CommunityEventHostCollectionData>
```

Needed for proper typing.